app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cachet = {}
+
+    def wrapper(*args) -> int | str:
```

Comment:
- What if the function takes key arguments?

- Also what if decorator is applied to two different functions that take the same arguments?

Reply:
Thank you for your corrections.
Fixed.